### DISCUSSION TITLE
Welcome to GSServer Discussions!

### DISCUSSION THREAD

**Discussion by rmorgan001** [rmorgan001/GSServer#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Astrobabb** [comment]
Hi, I may be in the wrong place here but hope not, looking for info / help with Green Swamp Software which has some affiliation with github.

I am into astro imaging and a necessary evil is software to allow my scope to slew to and center an object then maintain that positioning accuracy to within an arc second using equipment that doesn't cost five figures. Software also has to control my cameras.
The software to do this isn't written as a singular piece, at least three maybe four different proprietary software programs have to work simultaneously to provide the above function.
Currently I was using ASCOM / EQ-MOD / Stellarium / PHD2 / NINA. 
Last evening the chain of this work broke, my Stellarium software would no longer initiate EQ-MOD, and so my imaging hopes were dashed.
I had a few years ago over a period of time figured out how to make that chain work, with some help from a friend and the internet, however due to declining mental health I'm unsure if I can re-figure it out to effect repair.
Which brings me here, wondering if GSS is any easier to "figure out" or configure as a replacement for EQ-MOD, having seen a few references to that.

Any pointing to web sites, text, videos etc. that can give me an overview to GSS capabilities so I can determine if there is any benefit to it over what I was currently using would be good. Further should GSS be of interest to that end any web sites, text, videos etc. that could help me to integrate it into my work flow would be greatly appreciated.

As to hardware, I have a skywatcher EQ-6 Pro using an EQ-dir modem (DB-9 to USB) from Shoestring astronomy which takes the place of the hand controller in the communication path between a laptop and mount electronics. Also I have an EQ-8 but it is currently not in use, but will be later this spring / summer using a similar communication scheme but I am not seeking help with that mount at this time.

If I am posting this in the wrong place perhaps someone on your side could point me where I can post this help request?

Hoping to get some help.
Robert Babb

**Reply by rmorgan001** [reply]
Robert,  These two sites are good to start with... http://www.greenswamp.org/ for the videos and gss.groups.io for any support.
